uptrain/framework/evalllm.py

```diff
@@ -79,9 +80,9 @@ def evaluate(
                 req_attrs.update([schema.conversation])
 
             if isinstance(m, ParametricEval):
-                ser_checks.append({"check_name": m.__class__.__name__, **m.dict()})
+                ser_checks.append({"check_name": m.__class__.__name__, **m.dict(), "scenario_description": scenario_description})
             elif isinstance(m, Evals):
-                ser_checks.append(m.value)
+                ser_checks.append({"check_name": m.value, "scenario_description": scenario_description})
```

Comment:
Why not: 

**{"scenario_description": scenario_description}

Reply:
elif isinstance(m, Evals):
         dictn = {"scenario_description": scenario_description}
         ser_checks.append({"check_name": m.value, **dictn)